src/lib/operators/join_hash.cpp

```diff
@@ -46,9 +46,10 @@ const std::string& JoinHash::name() const {
 }
 
 std::string JoinHash::description(DescriptionMode description_mode) const {
+  const char separator = (description_mode == DescriptionMode::SingleLine ? ' ' : '\n');
   std::ostringstream stream;
-  stream << AbstractJoinOperator::description(description_mode);
-  stream << " Radix bits: " << (_radix_bits ? std::to_string(*_radix_bits) : "Unspecified");
+  stream << AbstractJoinOperator::description(description_mode) << separator;
+  stream << "Radix bits: " << (_radix_bits ? std::to_string(*_radix_bits) : "Unspecified");
```

Comment:
Radix bits became part of the tooltip a while ago. Since I am deeply invested in the join, I like the radix information. But it's probably irrelevant for most people.

@julianmenzler @Bensk1 : does it have a value to you listing the radix bits in the PQP? Or is the tooltip sufficient?

Reply:
For me, the tooltip would be sufficient.

Reply:
Same for me.

Reply:
Then feel free to remove the radix information from the node.